src/core/Akka/Routing/Router.cs

```diff
@@ -188,13 +188,14 @@ public class Router
         //So in order to not confuse the compiler we demand at least one ActorRef. /@hcanber
         public Router(RoutingLogic logic, IActorRef routee, params IActorRef[] routees)
         {
-            var routeesLength = routees.Length;
```

Comment:
If an explicit `null` is being passed as `routees`, the `.Length` will throw a NRE.